sschipke/palette-producer-api
# Hash and compare passwords in db

## What's this PR do?
Hashes users passwords when stored in the database
### Where should the reviewer start?
- app.js in `/signup` and `/login` routes
### How should this be manually tested?
Register a user, see that their password is hashed on the database
Then login the user and see if only the correct password works
